### ISSUE TITLE
Is there a way to globally adjust tree based on the node height? 

### ISSUE THREAD

**Issue by rchancey** [frontend-collective/react-sortable-tree#439]
I am using the title and sub title and being old I need larger fonts... and sometimes that pushes the title outside the bounds of the node. I can change the height of the node but then it throws off all the  connectors (they no longer stop at the node border edge).

Is there a way to set global hight and have the tree adapt?

**Comment by anil1712** [comment]
@rchancey did you get any solution for this?

**Comment by rchancey** [comment]
Nope.. I wish I had.. a very nice widget and quite powerful.. but custom styling sucks :-(
